NativeApp/src/services/holidayService.js

```diff
@@ -0,0 +1,5 @@
+import axios from '../utilities/AxiosInstance';
+
+export const getHolidays = employeeId => {
```

Comment:
not remotely an issue but I've noticed inconsistent use of brackets with single arg arrow functions.

Its worth picking one approach and enforcing in eslintrc file